apps/rush-lib/src/cli/logic/PackageChangeAnalyzer.ts

```diff
@@ -49,13 +50,24 @@ export class PackageChangeAnalyzer {
 
     const noProjectHashes: { [key: string]: string } = {};
 
-    // Load the package deps hash for the whole repository
-    const repoDeps: IPackageDeps = PackageChangeAnalyzer.getPackageDeps(
-      this._rushConfiguration.rushJsonFolder, [RushConstants.packageDepsFilename]
-    );
+    let repoDeps: IPackageDeps;
+    try {
+      // Load the package deps hash for the whole repository
+      repoDeps = PackageChangeAnalyzer.getPackageDeps(
+        this._rushConfiguration.rushJsonFolder + 'asdfasdfasdfasdfasdf', [RushConstants.packageDepsFilename]
+      );
+    } catch (e) {
```

Comment:
>catch (e) { [](start = 6, length = 11)

Is there a way to do this without an unexceptional exception?  For example, can we simply test whether we are in a Git repository, and disable all the Git operations if not?